### PR TITLE
Release 1.3.4

### PR DESCRIPTION
**Pull request by ggwpez**
Release for the updated inflation formula from https://github.com/polkadot-fellows/runtimes/pull/471

This release only affects the Polkadot runtime.

### CODE REVIEW

**Comment by ggwpez** [comment]
> So why bump and release all (unaffected) runtimes? We could bump and release only Polkadot Relay.

Yes true. We did partial bumps in the past, and it then becomes difficult to know what runtimes need to be bumped to what version.  
Bumping all to 1.3.4 in the 1.3.4 release keeps the rule simple - just bump all to the release version.

**Comment by acatangiu** [comment]
Releasing same runtime again but with a different spec version is not cool IMO. It adds unnecessary operational overhead to do useless runtime upgrades.

I suggest we achieve what you describe by bumping all **relevant** runtimes to latest spec version.

In this case, only Polkadot Relay gets 1.3.4. E.g. later say we want to release AH runtime, it would get 1.3.5 (and so will any other runtime that gets changes between now and then).

> difficult to know what runtimes need to be bumped to what version on the next release

"what runtimes" -> check the changelog or git history
"what version" -> always use next "globally unique" version (1.3.4 now, next 1.3.5 or 1.4.0)

**Comment by ggwpez** [comment]
> It adds unnecessary operational overhead to do useless runtime upgrades.

We dont upgrade them. They are just deleted from the release page. But it makes it simpler to ctrl+f update them all.

>"what version" -> always use next "globally unique" version (1.3.4 now, next 1.3.5 or 1.4.0)

Yea we can also do that. Will change now to get this merged.

**Review by ordian — APPROVED** [review]

**Review by seadanda — APPROVED** [review]

**Review by gpestana — APPROVED** [review]

**Review by muharem — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

**Review by seadanda — APPROVED** [review]
Do we need to re-run benchmarks for MQ bump? AHK and Encointer bumps LGTM

File: Cargo.lock [hunk 1]
```diff
@@ -8559,9 +8559,9 @@ dependencies = [
 
 [[package]]
 name = "pallet-message-queue"
-version = "39.0.0"
+version = "39.0.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "20e65a37881d1998546254a5e50a1f768b3f82deabe774e750f4ea95aba8030c"
+checksum = "8bb779dbfd58b3a4e6a84245392fe9d8c31a746ff7c6db414e91c012663859c3"
```
Comment (acatangiu) [inline Cargo.lock:8564]: To pick up this fix for other runtimes we need to bump them too. Not sure if the fix is urgent, if the other system chains have any overweight stuck messages in the queue.
But if they don't, I don't personally think it's worth the runtime upgrade, and we can go without bumping them.

Comment (ggwpez) [reply Cargo.lock:8564]: Good Q. I think it should be fine to only bump PAH. Last time we checked it was only Hydra affected (Astar was false positive).

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.3.4] 31.10.2024
```
Comment (seadanda) [inline CHANGELOG.md:7]:
```suggestion
## [1.3.4] 01.11.2024
```
WDYT about today? Or are there outstanding things to fix?

Comment (ggwpez) [reply CHANGELOG.md:7]: I was just a bit paranoid about the inflation change since its so significant. Checked it twice now again locally, should be good.

Comment (seadanda) [reply CHANGELOG.md]: yea fair enough!

**Pull request merged**